geopackage_validator/generate.py

```diff
@@ -37,27 +38,32 @@ def generate_table_definitions(
         if columns[0][2] and not projection:
             projection = columns[0][2]
 
+        if columns[0][0] and not column_name:
+            column_name = columns[0][0]
+
         info = dataset.ExecuteSQL(
             "PRAGMA TABLE_INFO('{table_name}');".format(table_name=table[0])
         )
 
-        columnlist = [Column(column_name=item[1], data_type=item[2]) for item in info]
-        columnlist.extend(
-            [
-                Column(column_name=column[0], geometry_type_name=column[1])
-                for column in columns
-            ]
+        column_list = [Column(column_name=item[1], data_type=item[2]) for item in info]
+        column_list.extend(
```

Comment:
lijkt me overbodig wan regel 48 zorgt er al voor dat alle kolommen in de columnlist komen, nee?